# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/valentin/kismet/pcap-bpf_location.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN kismet-2004-04-R1/pcapsource.h kismet-2004-04-R1-new/pcapsource.h
--- kismet-2004-04-R1/pcapsource.h	2004-03-21 00:57:04.000000000 +0100
+++ kismet-2004-04-R1-new/pcapsource.h	2004-05-01 12:02:50.683464560 +0200
@@ -34,7 +34,7 @@
 extern "C" {
 #ifndef HAVE_PCAPPCAP_H
 #include <pcap.h>
-#include <net/bpf.h>
+#include <pcap-bpf.h>
 #else
 #include <pcap/pcap.h>
 #include <pcap/net/bpf.h>
